connectors/sources/oracle.py

```diff
@@ -423,9 +435,11 @@ async def fetch_documents(self, table):
             Dict: Document to be indexed
         """
         try:
+            self._logger.info(f"Fetching records for the table: {table}")
             row_count = await self.oracle_client.get_table_row_count(table=table)
             if row_count > 0:
                 # Query to get the table's primary key
+                self._logger.debug(f"Total '{row_count}' rows found in '{table}' table")
```

Comment:
```suggestion
                self._logger.debug(f"Total {row_count} rows found in table \"{table}\"")
```

Reply:
@artem-shelkovnikov these changes are causing the linting error.

Reply:
It's not automatically fixable by `make autoformat`?

Reply:
I already tried to do that but it was reformatting the log to the previous one. It's not accepting the current logs you suggested.

Reply:
It cannot reformat to previous one, because I've changed wording of the log slightly?

Reply:
Does it complain at `\"`?

Reply:
Yes, it removes the escaping

Reply:
We can do that, it's okay, as long as it's consistent.